tests/phpunit/integration/Modules/Idea_HubTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Idea_HubTest
+ *
+ * @package   Google\Site_Kit\Tests\Modules
+ * @copyright 2021 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Modules;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Modules\Module_With_Scopes;
+use Google\Site_Kit\Core\Modules\Module_With_Settings;
+use Google\Site_Kit\Core\Storage\Options;
+use Google\Site_Kit\Modules\Idea_Hub\Settings;
+use Google\Site_Kit\Modules\Idea_Hub;
+use Google\Site_Kit\Tests\Core\Modules\Module_With_Settings_ContractTests;
+use Google\Site_Kit\Tests\TestCase;
+
+/**
+ * @group Modules
+ */
+class Idea_HubTest extends TestCase {
+	use Module_With_Settings_ContractTests;
+
+	public function test_register() {
+		$analytics = new Idea_Hub( new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE ) );
```

Comment:
Please make sure variables are updated accordingly 😄

Reply:
Updated in https://github.com/google/site-kit-wp/pull/3223/commits/3f6a2d3795733ef1ad792ca5bcd4f0cabec88308